### PR TITLE
improve in order key generation

### PR DESCRIPTION
**Pull request by xiang90**
Generating in order key that padding with 0 as its prefix.

It would be help to sort them lexicographically.

### CODE REVIEW

**Comment by xiang90** [comment]
Note this breaks the API and previous behavior. I *think* it might be OK, since people should not rely one the naming (if they do, they already have done it wrong... they has to compare the created index to sort anyway)

File: Documentation/api.md [hunk 1]
```diff
@@ -423,13 +423,13 @@ curl -s 'http://127.0.0.1:2379/v2/keys/queue?recursive=true&sorted=true'
         "nodes": [
             {
                 "createdIndex": 2,
-                "key": "/queue/2",
+                "key": "/queue/00000000000000000002",
```
Comment (yichengq) [inline Documentation/api.md]: i think we need to update the key field in create response too.

**Comment by yichengq** [comment]
I agree that this change still provides the original guarantee.

File: Documentation/api.md (hunk 1 above)
Comment (xiang90) [inline Documentation/api.md]: good catch. i will change.

**Comment by xiang90** [comment]
@yichengq Updated. PTAL.

**Comment by yichengq** [comment]
lgtm

**Pull request merged**